AutomatedTesting/Gem/PythonTests/Physics/tests/collider/Collider_PxMeshNotAutoAssignedWhenNoPhysicsFbx.py

```diff
@@ -15,7 +15,7 @@ class Tests():
     create_entity        = ("Created test entity",                   "Failed to create test entity")
     mesh_added           = ("Added Mesh component",                  "Failed to add Mesh component")
     physx_collider_added = ("Added PhysX Collider component",        "Failed to add PhysX Collider component")
-    assign_mesh_asset    = ("Assigned Mesh asset to Mesh component", "Failed to assign mesh asset to Mesh component")
+    assign_model_asset    = ("Assigned Model asset to Mesh component", "Failed to assign model asset to Mesh component")
```

Comment:
again their non-PEP8 alignment is off for extra space... this isn't really an issue just style compliance for their code